ui/src/app/dashboard/total-summary/total-summary.component.css

```diff
@@ -0,0 +1,23 @@
+* {
+  font-family: 'Roboto', sans-serif;
+}
+
+.mat-card {
+  float: left;
+  margin: 1rem;
+}
+
+.count a {
+  text-decoration: none;
+  color: #000000;
+}
+
+.mat-card-title {
+  font-size: 1.2rem;
+  color: gray
+}
+
+/*TODO(zachx): change this to adapt more complex layout*/
+:host {
```

Comment:
This is an experimental selector which isn't supported in Safari according to mdn. I might suggest putting a container div around the whole thing since you're likely to need that anyways for styling.